engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/DefaultResponseEncoder.scala

```diff
@@ -1,13 +1,19 @@
 package pl.touk.nussknacker.engine.requestresponse
 
 import io.circe.Json
+import org.everit.json.schema.Schema
 import pl.touk.nussknacker.engine.requestresponse.api.ResponseEncoder
-import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+import pl.touk.nussknacker.engine.schemedkafka.encode.{BestEffortJsonSchemaEncoder, ValidationMode}
 
 object DefaultResponseEncoder extends ResponseEncoder[Any] {
 
-  private val bestEffortEncoder = BestEffortJsonEncoder(failOnUnkown = true, getClass.getClassLoader)
+  private val bestEffortEncoder = new BestEffortJsonSchemaEncoder(ValidationMode.lax)
 
-  override def toJsonResponse(input: Any, result: List[Any]): Json = bestEffortEncoder.encode(result)
+  override def toJsonResponse(input: Any, result: List[Any], schema: Option[Schema]): Json = {
```

Comment:
I somehow think that schema should be part of ResponseEncoder. Arguments to `toJsonResponse` are only known at runtime, while schema is know at (scenario) compileTIme